doc/manual/metaprogramming.rst

```diff
@@ -144,13 +144,13 @@ representations together:
 
```

Comment:
Missed the rename of ``:func:`symbol` `` here. It might also be more intuitive to rewrite it and call it the `Symbol` constructor rather than a regular function.